app/components/works/edit/work_subtypes_component.html.erb

```diff
@@ -3,6 +3,8 @@
     <%= content %>
   <% else %>
     <%= render Elements::Forms::FieldsetComponent.new(label: t('works.edit.fields.work_subtypes.label')) do %>
+      <%= render Elements::Forms::InvalidFeedbackComponent.new(form:, field_name: error_field_name) if error_field_name %>
```

Comment:
This allows providing an error that is specific to a work subtype instead of using the same error message for all work subtypes.

Reply:
Would it have unintended consequences if we removed the `if error_field_name` here and changed [`InvalidFeedbackComponent#render?`](https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/components/elements/forms/invalid_feedback_component.rb#L23) to:

```ruby
      def render?
        errors.present? && field_name.present?
      end
```

(I assume it *would* have bad side effects, but it's worth asking if we can use VC's built-in conditional rendering semantics here.)

Reply:
That'll work. Fixing.